draft-packages/guidance-block/docs/GuidanceBlock.stories.tsx

```diff
@@ -98,6 +101,10 @@ DefaultStory.argTypes = {
     description:
       "This takes a scene scene or spot element, ie: `<Informative />`. This radio button implementation is a storybook only representation to toggle between the two illustration styles.",
   },
+  content: {
+    description:
+      "If you need to render custom content insight to the `GuidanceBlock` that is more than just a title and description use this prop instead of the default `text` option.",
```

Comment:
I think you meant to write this, didn't you?
```suggestion
      "If you need to render custom content inside of the `GuidanceBlock` that is more than just a title and description use this prop instead of the default `text` option.",
```